app/models/miq_schedule_worker/jobs.rb

```diff
@@ -46,9 +46,21 @@ def job_check_for_evm_snapshots(job_not_found_delay)
     queue_work_on_each_zone(:class_name  => "Job", :method_name => "check_for_evm_snapshots", :args => [job_not_found_delay])
   end
 
-  def job_proxy_dispatcher_dispatch
-    if JobProxyDispatcher.waiting?
-      queue_work_on_each_zone(:class_name => "JobProxyDispatcher", :method_name => "dispatch", :task_id => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY, :role => "smartstate")
+  def vm_scan_dispatcher_dispatch
+    if VmScan::Dispatcher.waiting?
+      queue_work_on_each_zone(:class_name => "VmScan::Dispatcher", :method_name => "dispatch", :task_id => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY, :role => "smartstate")
+    end
+  end
+
+  def container_scan_dispatcher_dispatch
+    if ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::Dispatcher.waiting?
+      queue_work_on_each_zone(:class_name => "ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::Dispatcher", :method_name => "dispatch", :task_id => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY, :role => "smartstate")
+    end
+  end
+
+  def infra_conversion_dispatcher_dispatch
+    if InfraConversionJob::Dispatcher.waiting?
+      queue_work_on_each_zone(:class_name => "InfraConversionJob::Dispatcher", :method_name => "dispatch", :task_id => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY)
```

Comment:
Does this need a role?  The others have smartstate.

Reply:
The job proxy dispatcher running as role smartstate was a holdover from primarily dispatching VmScan jobs, the v2v dispatcher is fine just running on the default/generic role